assets/app/views/_project-nav.html

```diff
@@ -16,21 +16,10 @@
       <div class="active-filters"></div>
     </div>
     <div class="col-sm-3 create-button-container">
-      <span ng-if="project.status.phase == 'Active'">
-        <a href="project/{{projectName}}/create" class="btn btn-lg btn-primary">
-          <span>Create</span>
-          <i class="fa fa-plus" />
-        </a>
-      </span>
-      <span ng-if="project.status.phase != 'Active'">
-        <a href="project/{{projectName}}/create" class="btn btn-lg btn-primary" disabled="disabled">
-          <span>Create</span>
-          <i class="fa fa-plus" />
-        </a>
-        <span ng-if="project.status.phase == 'Terminating'" class="pficon-layered" data-toggle="tooltip" data-placement="left" title="This project has been marked for deletion. No content may be created in this project." style="cursor: help; font-size: 32px; vertical-align: top; margin-left: 5px;">
-          <span class="pficon pficon-warning-triangle"></span>
-          <span class="pficon pficon-warning-exclamation"></span>
-        </span>
+      <a href="project/{{projectName}}/create" ng-disabled="project.status.phase != 'Active'" class="btn btn-lg btn-primary">Create...</a>
```

Comment:
the `project.status.phase != 'Active'` check was there to prevent a flash from appearing before project had been loaded